browser/ui/views/brave_actions/brave_actions_container.cc

```diff
@@ -110,7 +122,12 @@ void BraveActionsContainer::Init() {
           brave_shields::features::kBraveShieldsPanelV1)) {
     actions_[brave_extension_id].position_ = 1;
   }
-  actions_[brave_rewards_extension_id].position_ = ACTION_ANY_POSITION;
+  if (UsingRewardsExtension()) {
+    brave_rewards::RewardsPanelCoordinator::RegisterForBrowser(browser_);
```

Comment:
Is this the correct place to ensure that the RewardsPanelCoordinator is created for the current browser? Just feels like ui/views isn't the best place to make sure the extension is loaded, etc.

Reply:
I put these calls to create the `RewardsPanelCoordinator` in the views classes just because it was closest to the handling and because I didn't know of a better place to put it. In Chromium, `BrowserUserData` is only used right now for the side panel code, and those classes ultimately get instantiated from the `BrowserView` constructor. Would the constructor for `BraveBrowserView` be more appropriate?

Reply:
Well, `SidePanelCoordinator` is in `browser/ui/views` and only referenced from other places in Views, so I think that's why it's more ok for its "make sure you're started up" call to also be there. `RewardsPanelCoordinator` is in `brave/browser/brave_rewards`, and theoretically could be used by non-Views UI (android, etc), so maybe something like `BraveBrowser`?

Reply:
It turns out that the constructor body for `BraveBrowser` runs too late (after the constructor for `BraveRewardsActionView`. I've added the registration call to `BraveBrowserView` instead.